From: Torsten Werner <twerner@debian.org>
Date: Sun, 15 Aug 2010 00:02:18 +0200
Subject: do not build InvokeDynamicSupport.java

... because it requires a dependency from the Java7 world which is not packaged
yet.
---
 build.xml |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/build.xml
+++ b/build.xml
@@ -57,6 +57,7 @@
     <include name="**/*.java"/>
     <exclude unless="bsf.present" name="org/jruby/javasupport/bsf/**/*.java"/>
     <exclude unless="sun-misc-signal" name="**/SunSignalFacade.java"/>
+    <exclude name="org/jruby/runtime/invokedynamic/InvokeDynamicSupport.java"/>
   </patternset>
 
   <patternset id="ruby.src.pattern">
